### ISSUE TITLE
Add tests to GitHub actions

### ISSUE THREAD

**Issue by andycraig** [Ikuyadeu/vscode-R#318]
Once the test works, we could add tests to GitHub actions so that it could run regularly.

_Originally posted by @renkun-ken in https://github.com/Ikuyadeu/vscode-R/pull/314#issuecomment-624951920_

**Comment by andycraig** [comment]
This commit looks like an example of this: https://github.com/microsoft/vscode-test/pull/54/commits/c51eedf2e66084d1e83dd63589c24a4761f5528f

**Comment by renkun-ken** [comment]
Yes, I've done this at https://github.com/renkun-ken/vscode-R/blob/migrate-vscode-test/.github/workflows/main.yml.

It seems to make sense I file a separate PR for this once #317 is merged.

**Comment by andycraig** [comment]
Done in #317. Thanks @renkun-ken!
